runtime/src/main/scala/akka/grpc/GrpcProtocol.scala

```diff
@@ -34,7 +34,7 @@ trait GrpcProtocol {
    * @param codec the compression codec to encode data frame bodies with.
    */
   @InternalApi
-  def newWriter(codec: Codec): GrpcProtocolWriter
+  private[grpc] def newWriter(codec: Codec): GrpcProtocolWriter
```

Comment:
Actually having this public-but-marked-InternalApi came in handy at https://github.com/cloudstateio/cloudstate/pull/211/files#diff-738f4776d7ff0cc32b60cc61f1676218R444 - might be OK to keep it?

Reply:
Should we annotate as `InternalStableApi` then?

Reply:
Good question. The cloudstate proxy is not a library that gets included in downstream projects, but a 'leaf' application. That means there is no reason cloudstate would have a different version of Akka gRPC on its classpath than the one it directly depends on. That means if we make any changes to Akka gRPC, there is no risk of binary incompatibilities in cloudstate: when cloudstate updates to a newer version of Akka gRPC, we can just fix the compile errors in that PR.

So for that reason `InternalStableApi` is less important here. On the other hand, it might still be useful to somehow signal that we don't want to make these API's entirely inaccessible because they have a use case there... I'm thinking perhaps a comment, but I'm not sure what that comment should say :).

Reply:
Hmmm, I'd rather use the annotation than a comment (on top of the `INTERNAL API`).

I think the cloudstate use case fits quite well the description in [InternalStableApi](https://github.com/akka/akka/blob/master/akka-actor/src/main/java/akka/annotation/InternalStableApi.java).

Reply:
OK!

Reply:
I mean, I read that description and think of `cinnamon` or Lagom and not so much about cross-library compat.